docs/contribute/community.rst

```diff
@@ -6,41 +6,22 @@
     :description: Find the CobraTeam and Splinter communities.
     :keywords: splinter, python, cobrateam, community, atdd, tests, acceptance tests, web applications
 
-+++++++++
-Community
-+++++++++
++++++++++++++++++++
+Community Resources
++++++++++++++++++++
 
-mailing list
+Mailing List
 ============
 
 * `splinter-users <http://groups.google.com/group/splinter-users>`_ - list for help and announcements
 * `splinter-developers <http://groups.google.com/group/splinter-developers>`_ - where the developers of splinter itself discuss new features
 
-irc channel
+IRC channel
 ===========
 
 #cobrateam channel on irc.freenode.net - chat with other splinter users and developers
 
-ticket system
+Ticket System
```

Comment:
I'd change the title of this section to "Issue tracker" or something like that. "Ticket system" is something I haven't heard in a while lol

Reply:
👍